tasks/yesno.py

```diff
@@ -0,0 +1,301 @@
+
```

Comment:
On Sun, Apr 24, 2016 at 02:23:57AM -0700, Silvicek wrote:
> > @@ -0,0 +1,301 @@
> > +
> 
> The hypev task learns on separate samples and the averaging is done only when evaluating. Load_hypev is therefore unusable in yesno. In my opinion the hypev task is outdated..

I don't understand why does the averaging imply that load_hypev() is
unusable, can you elaborate?

What we need to show in any papers we publish is that our method has
merit compared to a simpler averaging baseline.